Mod Manager/src/routes/modList/+page.svelte

```diff
@@ -550,24 +644,83 @@
 <Modal passiveModal open={frameworkDeployModalOpen} modalHeading="Applying your mods" preventCloseOnClickOutside>
 	Your mods are being deployed. This may take a while - grab a coffee or something.
 	<br />
-	<pre
-		class="mt-2 h-[10vh] overflow-y-auto whitespace-pre-wrap bg-neutral-800 p-2"
-		style="font-family: 'Fira Code', 'IBM Plex Mono', 'Menlo', 'DejaVu Sans Mono', 'Bitstream Vera Sans Mono', Courier, monospace; color-scheme: dark"
-		id="deployOutputElement">{@html deployOutputHTML}</pre>
-	{#if deployOutput.split(/\r?\n/).some((a) => a.match(/.*WARN.*?\t/)) || deployOutput.split(/\r?\n/).some((a) => a.match(/.*ERROR.*?\t/))}
-		<br />
-		<div class="flex flex-row gap-2 flex-wrap max-h-[15vh] overflow-y-auto">
-			{#each deployDiagnostics as line}
-				<InlineNotification hideCloseButton lowContrast kind={line.includes("WARN") ? "warning" : "error"}>
-					<div slot="title" class="-mt-1 text-lg">
-						{line.includes("WARN") ? "Warning" : "Error"}
-					</div>
-					<div slot="subtitle">{line.replace(/.*WARN.*?\t/, "").replace(/.*ERROR.*?\t/, "")}</div>
-				</InlineNotification>
-			{/each}
+    <div 
+		id="deployOutputElement"
+		class="mt-2 h-[40vh] overflow-y-auto bg-neutral-800 p-2"
+        style="font-family: 'Fira Code', 'IBM Plex Mono', 'Menlo', 'DejaVu Sans Mono', 'Bitstream Vera Sans Mono', Courier, monospace; color-scheme: dark"
+		on:scroll={(event) => handleScroll(event, 'deployOutputElement')}>
+
+        {#each outputLines as line}
+            <pre class="whitespace-pre-wrap break-words text-gray-600">{@html convertAnsi.toHtml(line)}</pre>
+        {/each}
+
+        {#if userHasScrolledDeployOutput}
+            <button 
+                class="absolute top-[288px] z-10 right-[37px] bg-gray-400 hover:bg-red-500 text-white py-1 px-3 rounded"
+                on:click={enableAutoScrollDeployOutput}>
+                <svg width="24px" height="24px" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M4 9l8 8 8-8" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+                </svg>
+            </button>
+        {/if}
+
+    </div>
+    {#if warnings.length > 0 || errors.length > 0}
+        <br />
+        <div id="notificationElement"
+			class="flex flex-row gap-2 flex-wrap max-h-[25vh] h-[126px] overflow-y-auto relative" 
```

Comment:
Same thing as before with this absolute height value; usually Tailwind values are preferable as well (h-40 or something; Tailwind autocomplete will tell you the pixel values)